### PR TITLE
Install torchaudio from Conda

### PR DESCRIPTION
**Pull request by rosbo**
BUG=143372552

### CODE REVIEW

**Comment by rosbo** [comment]
This is failing with:

```
ImportError: Failed to import test module: test_torchaudio
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/opt/conda/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/input/tests/test_torchaudio.py", line 3, in <module>
    import torchaudio
  File "/opt/conda/lib/python3.6/site-packages/torchaudio/__init__.py", line 5, in <module>
    import _torch_sox
ImportError: /opt/conda/lib/python3.6/site-packages/_torch_sox.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at11ATenOpTable11reportErrorEN3c1012TensorTypeIdE
```

**Comment by rosbo** [comment]
After #663, tests are now passing.

File: Dockerfile [hunk 1]
```diff
@@ -180,15 +180,9 @@ RUN pip install scipy && \
     # PyTorch
     export CXXFLAGS="-std=c++11" && \
     export CFLAGS="-std=c99" && \
-    conda install -y pytorch torchvision cpuonly -c pytorch && \
-    # PyTorch Audio
     apt-get install -y sox libsox-dev libsox-fmt-all && \
```
Comment (mcollins42) [inline Dockerfile]: Nit: I think that we don't need to set complier flags or install sox and cffi anymore unless they are needed for something else.

Comment (rosbo) [reply Dockerfile]: Thanks for flagging. I will try removing them in a separate PR.

**Pull request merged**